src/actions/login.ts

```diff
@@ -0,0 +1,41 @@
+"use server";
+import { signIn } from "@/auth";
+import { loginSchema } from "@/components/Login/schemas";
```

Comment:
what do you think about moving schemas which are used both in forms and sever actions to global schemas folder in src?

Reply:
If it will prevent code repetition of the same code - sure

Reply:
i thought about it too earlier, i would move it to schemas dir